### PR TITLE
feat(wasm): add `beerus-web` wasm library

### PR DESCRIPTION
**Pull request by sergey-melnychuk**
This PR brings proper WebAssembly support to Beerus.

The demo web page is added in order to showcase expected usage.

Reference: 
- https://github.com/eigerco/beerus/pull/691
- https://github.com/sergey-melnychuk/helios/commits/beerus-wasm/
- https://github.com/sergey-melnychuk/sequencer/commits/beerus-wasm/
- https://github.com/sergey-melnychuk/cairo-vm/commits/beerus-wasm/
- https://github.com/sergey-melnychuk/cairo/commits/beerus-wasm/
- https://github.com/eigerco/beerus/pull/754

Closes https://github.com/eigerco/beerus/issues/692 
Closes https://github.com/eigerco/beerus/issues/674

### CODE REVIEW

**Review by ICavlek — CHANGES_REQUESTED** [review]
I have added couple of comments, but in general excellent job :) I have managed to run it from my side and I had no issues :)

File: web/etc/proxy/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1235 @@
+{
+    "name": "proxy",
+    "version": "1.0.0",
+    "lockfileVersion": 3,
+    "requires": true,
+    "packages": {
+        "": {
+            "name": "proxy",
+            "version": "1.0.0",
+            "license": "ISC",
```
Comment (ICavlek) [inline web/etc/proxy/package-lock.json:10]: Please check also regarding this license value when everything has been compiled

**Review by ICavlek — APPROVED** [review]
LGTM, please also check the package-lock.json, license value

**Review by LKozlowski — APPROVED** [review]
I went through the PR commit by commit and got the general idea behind that PR. It's a huge PR and as you said there are still a lot of rough edges that need to be polished in the future, but as for the general it seems very easy to follow. I haven't found anything particularly odd that needs to be fixed now. I mostly checked the rust code and skimmed through JS, as I assume this is just for demo purposes for now and will be heavily changed in the future. I think we should merge it as soon as possible and set up a job that will test the wasm code regularly. 
The most important thing is that it is working, and the details of it can be ironed out very easily.

**Pull request merged**